### PR TITLE
Small fixes and improvements

### PR DESCRIPTION
**Pull request by poncovka**
I was able to succesfully run all kickstart tests except for these:
* lvm tests fail with a kernel warning "possible circular locking dependency detected"
* proxy-auth fails to set up the installation source
* packages-excludedocs fails because there are files in /usr/share/doc/xz/
* nfs-repo-and-addon, ibft, iscsi

### CODE REVIEW

File: scripts/kstest-runner [hunk 1]
```diff
@@ -51,7 +51,7 @@ class VirtualInstall(object):
     Run virt-install using an iso and a kickstart
```
Comment (jkonecny12) [inline scripts/kstest-runner]: Please add a commit message why you are changing `-r` to `--memory`

File: screen-access-advanced.ks.in [hunk 2]
```diff
@@ -48,7 +48,7 @@ echo "${generated_config}" > etc/sysconfig/anaconda
 %post --nochroot --interpreter /bin/bash
```
Comment (jkonecny12) [inline screen-access-advanced.ks.in]: Please describe what you are fixing here in a few words in the commit message.

Comment (poncovka) [reply screen-access-advanced.ks.in]: Fixed.

File: scripts/kstest-runner (hunk 1 above)
Comment (poncovka) [reply scripts/kstest-runner]: Fixed.

**Pull request merged**